app/models/manageiq/providers/amazon/inventory/parser/storage_manager/ebs.rb

```diff
@@ -76,14 +76,11 @@ def link_volume_to_disk(volume_hash, attachments)
 
       dev = File.basename(a['device'])
 
-      disk_hash = {
```

Comment:
yes, the parser is loading existing InventoryObject and modifies them, so we need the find_or_build for that. It's not an issue when the parser runs alone, then it build it.